src/app/xapian/searchservice.ts

```diff
@@ -528,8 +528,6 @@ export class SearchService {
             progressSnackBar.instance.messagetextsubject.next(getProgressSnackBarMessageText());
           }
 
-          this.rmmapi.deleteFromMessageContentsCache(this.pendingMessagesToProcess[this.processMessageIndex].id);
```

Comment:
The lack of this seems to break us, since changing message flags no longer invalidates the cache.